<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dishDetailUI</class>
 <widget class="QWidget" name="dishDetailUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>121</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>labels</string>
   </property>
  </widget>
  <widget class="QLabel" name="detailLabel">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>231</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>detail</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceText">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>69</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>price</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
